channels/applicationinsights-channel-js/Tests/Unit/src/Sender.tests.ts

```diff
@@ -1280,7 +1280,6 @@ export class SenderTests extends AITestClass {
                 let appInsightsCore = new AppInsightsCore();
                 appInsightsCore.logger = new DiagnosticLogger();
                 let messageId: _InternalMessageId = _InternalMessageId.InvalidInstrumentationKey;
-                let logInternalSpy = this.sandbox.spy(appInsightsCore.logger, 'logInternalMessage');
```

Comment:
Removed as the internal workings of the logger no longer calls the "public" method (part of the optimization)